pkg/swagger/typewalker.go

```diff
@@ -151,6 +151,18 @@ func (tw *typeWalker) _define(definitions Definitions, t *types.Named) {
 	s.Description = strings.Trim(path[len(path)-2].(*ast.GenDecl).Doc.Text(), "\n")
 	s.Enum = tw.enums[t]
 
+	// Enum extensions allows non-breaking api changes
+	// https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-enum
+	c := strings.Split(t.String(), ".")
+	name := c[(len(c) - 1)]
+	switch name {
+	case "VMSize":
```

Comment:
Don't hard-code this here.  Find a way to pass this in from swagger.go.

Reply:
| Suggest we stop generating the 2019-12-31-preview swagger now.

Separate PR